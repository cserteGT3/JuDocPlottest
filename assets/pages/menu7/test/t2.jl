# This file was generated, do not modify it. # hide
import PlotlyJS
trace1 = PlotlyJS.scatter(;x=1:4, y=[10, 15, 13, 17], mode="markers")
trace2 = PlotlyJS.scatter(;x=2:5, y=[16, 5, 11, 9], mode="lines")
trace3 = PlotlyJS.scatter(;x=1:4, y=[12, 9, 15, 12], mode="lines+markers")
p = PlotlyJS.plot([trace1, trace2])
#p = plot([trace1, trace2, trace3])
#relayout!(p, margin=attr(t=60, b=60, l=50, r=50))
println("""~~~
  <div id=test2 style=width:600px;height:250px;></div>""")
println(PlotlyJS.html_body(p.plot))
println("~~~")