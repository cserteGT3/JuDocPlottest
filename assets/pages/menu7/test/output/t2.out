~~~
<div id=test2 style=width:600px;height:250px;></div>
<script>
    gd = (function() {
  var WIDTH_IN_PERCENT_OF_PARENT = 100;
  var HEIGHT_IN_PERCENT_OF_PARENT = 100;
  var gd = Plotly.d3.select('body')
    .append('div').attr("id", "79041ef8-e414-4fc9-b92b-97f56124f919")
    .style({
      width: WIDTH_IN_PERCENT_OF_PARENT + '%',
      'margin-left': (100 - WIDTH_IN_PERCENT_OF_PARENT) / 2 + '%',
      height: HEIGHT_IN_PERCENT_OF_PARENT + 'vh',
      'margin-top': (100 - HEIGHT_IN_PERCENT_OF_PARENT) / 2 + 'vh'
    })
    .node();
  var plot_json = {"layout":{"margin":{"l":50,"b":50,"r":50,"t":60}},"data":[{"mode":"markers","y":[10,15,13,17],"type":"scatter","x":[1,2,3,4]},{"mode":"lines","y":[16,5,11,9],"type":"scatter","x":[2,3,4,5]}]};
  var data = plot_json.data;
  var layout = plot_json.layout;
  Plotly.newPlot(gd, data, layout);
  window.onresize = function() {
    Plotly.Plots.resize(gd);
  };
  return gd;
})();

 </script>

~~~
